node-disk-manager.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: node-disk-manager
+spec:
+  template:
+    metadata:
+      labels:
+        app: node-disk-manager
+    spec:
+      nodeSelector:
+        nodeGroup: NDM-StorageNode
+      containers:
+      - name: node-disk-manager
+        command:
+        - /usr/sbin/ndm
+        - start
+        image: openebs/node-disk-manager:latest
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: device
+          mountPath: /host/dev
+        - name: mount
+          mountPath: /host/mnt
+          mountPropagation: Bidirectional
+        - name: etc
+          mountPath: /host/etc
+          mountPropagation: Bidirectional
+      volumes:
+      - name: device
+        hostPath:
+          path: /dev
+          type: Directory
+      - name: mount
+        hostPath:
+          path: /mnt
+          type: Directory
+      - name: etc
+        hostPath:
+          path: /etc
```

Comment:
Please experiment by sharing mtab or fstab files and check if it works, instead of sharing whole /etc

Reply:
@gkGaneshR I did what you suggested. Thanks a lot for your help.